lib/ruumba/analyzer.rb

```diff
@@ -61,12 +61,50 @@ def copy_erb_files(fq_dir, tmp, pwd)
       end
     end
 
+    def autofix!(target, tmp)
+      src_dir = Pathname.new(File.expand_path(target))
+
+      Dir.foreach(tmp) do |f|
+        # Match the extracted code from the tmp directory
+        # to its original file in the source directory in
+        # order to find-and-replace for --auto-correct.
+        src = f.split('.').tap(&:pop).join '.'
+
+        if File.extname(src) == '.erb' && File.exist?("#{src_dir}/#{src}")
+          # Walk through the extracted .erb.rb file and write
+          # its contents into the interpolation areas of the
+          # original .erb file.
+          code       = File.readlines("#{tmp}/#{f}")
+          noncode    = File.readlines("#{src_dir}/#{src}")
+                           .reject { |l| l =~ ERB_REGEX }
+          to_correct = File.readlines("#{src_dir}/#{src}")
+          out        = []
+
+          to_correct.each do |line|
+            # @TODO: This is pretty hacky. (EQW 30 Aug 2017)
+            out << if line.match?(/<%/)
+                     "  <% #{code.shift.rstrip} %>\n"
+                   elsif /<%=/
```

Comment:
This should be:

```rb
elsif line.match?(/<%=/)
```

Reply:
@SijiaDavis This could be the source of your trouble; I'll push up a correction tonight or tomorrow.

Reply:
Nice catch 👍🏽, unfortunately I don't think this is the source of why it wasn't working for me... I have some `<% ... %>` in my erb files that were supposed to be corrected but weren't...